Scripts/Runtime/Entities/Lifecycle/Spawner/EntitySpawner.cs

```diff
@@ -11,10 +11,19 @@
 namespace Anvil.Unity.DOTS.Entities
 {
     [UsedImplicitly]
-    internal class EntitySpawner<TEntitySpawnDefinition> : AbstractAnvilBase,
-                                                           IEntitySpawner
+    public class EntitySpawner<TEntitySpawnDefinition> : AbstractAnvilBase,
+                                                         ISharedWriteAccessControlledValue<EntitySpawnWriter<TEntitySpawnDefinition>>,
+                                                         IEntitySpawner
         where TEntitySpawnDefinition : unmanaged, IEntitySpawnDefinition
     {
+        private event Action<IEntitySpawner> OnPendingWorkAdded;
+
+        event Action<IEntitySpawner> IEntitySpawner.OnPendingWorkAdded
+        {
+            add => this.OnPendingWorkAdded += value;
+            remove => this.OnPendingWorkAdded -= value;
+        }
```

Comment:
```suggestion
        internal event Action<IEntitySpawner> IEntitySpawner.OnPendingWorkAdded
        {
            add => this.OnPendingWorkAdded += value;
            remove => this.OnPendingWorkAdded -= value;
        }
```

Reply:
> The modifier 'internal' is not valid for explicit interface implementation

It is implicitly internal because the interface is internal.

Reply:
Ahhh right, I forgot this was an explicit interface function.